### PR TITLE
Add feature to ban the participants for the inappropriate behaviour

### PR DESCRIPTION
**Pull request by Sanji515**
- Added the field `banned_email_ids` in Challenge model.

- Restricted the user to participate and make submissions for the challenge.

- And also removed the leaderboard entry for the particular participant team.

### CODE REVIEW

File: apps/jobs/views.py [hunk 1]
```diff
@@ -206,6 +206,14 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
             }
             return Response(response_data, status=status.HTTP_403_FORBIDDEN)
 
+        all_participants_email = participant_team.get_all_participants_email()
+        for i in all_participants_email:
```
Comment (RishabhJain2018) [inline apps/jobs/views.py]: `i` -> `participant_email`

File: apps/jobs/views.py [hunk 2]
```diff
@@ -206,6 +206,14 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
...
+            if i in challenge.banned_email_ids:
```
Comment (RishabhJain2018) [inline apps/jobs/views.py]: `i` -> `participant_email`

File: apps/jobs/views.py [hunk 3]
```diff
@@ -206,6 +206,14 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
...
+                response_data = {
+                    "error": "Sorry, You cannot make submissions in this challenge due to the inappropriate behaviour"
```
Comment (RishabhJain2018) [inline apps/jobs/views.py]: Please use the above error message.

File: apps/jobs/views.py (hunk 1 above)
Comment (Sanji515) [reply apps/jobs/views.py]: @RishabhJain2018 done

File: apps/jobs/views.py (hunk 2 above)
Comment (Sanji515) [reply apps/jobs/views.py]: done

File: apps/jobs/views.py (hunk 3 above)
Comment (Sanji515) [reply apps/jobs/views.py]: done

**Pull request merged**